src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -1569,15 +1569,13 @@ public void DisableSearchTextBox()
         }
 
         /// <summary>
-        /// Clear after closing down
+        /// Only reset the default values for elements associated with the view
         /// </summary>
-        internal void Close()
+        internal void PackageManagerViewClose()
         {
             TimedOut = false;   // reset the timedout screen 
             InitialResultsLoaded = false;   // reset the loading screen settings
-            RequestShowFileDialog -= OnRequestShowFileDialog;
-            nonHostFilter.ForEach(f => f.PropertyChanged -= filter_PropertyChanged);
-            LuceneUtility.DisposeAll();
```

Comment:
@QilongTang @reddyashish I think you should take a look at this. Should this be called from somewhere else, in this case @dnenov found that calling DisposeAll caused an access violation the next time you try to start the PackageManager View.

Instead of just being removed, should this be moved to shutdown/dispose of the PackageManagerViewExtension or DynamoView? IE somewhere higher up than the PackageManagerSearchView.

I also want to make sure we're not now leaking indexing data if we open and close the package manager search view over and over.

Reply:
This method was not meant to be a Dispose method for the PackageManagerSearchViewModel, and I am sorry that I might have mislead someone with the naming. The PackageManagerSearchViewModel is not being destroyed once created inside the DynamoView .. I suppose it is garbage collected once Dynamo closes? If LuceneUtility needs to be disposed of .. I don't know, I couldn't figure out what exactly it is disposing of, I only know that if it was disposed then, on subsequent PM view creation, it would crash.

Reply:
yes, I understand that this is not the dispose method, but it seems someone thought it was, I'm not sure who added this lucene dispose call, so I want to make sure that we don't need to find another place to relocate it to.